SUPPORTED-NETWORKS.md

```diff
@@ -0,0 +1,35 @@
+# Firehose Ethereum Instrumentation
+
+Firehose provides TWO mechanisms to extract data from an ethereum-like blockchain to `sf.ethereum.type.v2.Block` message:
+
+1. Running an instrumented node (data is extracted while the EVM actually executes the transactions) -> produces EXTENDED blocks
+2. Polling an standard node with a series of RPC calls (ex: get_block) -> produces BASE blocks
```

Comment:
```suggestion
1. Polling an standard node with a series of RPC calls (ex: get_block) -> produces BASE blocks
```

It's preferable (at least in my opinion) to always use `1.` and let the template generator generates the numbers.